### PR TITLE
fix: incorrect login ui

### PR DESCRIPTION
**Pull request by nhannt1311**

### CODE REVIEW

File: src/layouts/RegistrationForm.tsx
Comment (HumanLibrary) [inline src/layouts/RegistrationForm.tsx:1]: My push error for catching other than email exist, don't change it

Comment (nhannt1311) [reply src/layouts/RegistrationForm.tsx:1]: Yes, I roll-back it

**Comment by HumanLibrary** [comment]
This MR addressed right functionality and looks solid to merge.

**Pull request merged**